spec/requests/posts_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Your tests are missing to check `If a correct template was rendered`. Please make sure to include this check for all of your controllers.
  - Hint:  `it 'should render a template' do expect(response).to render_template('something') end`